<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户端（8080）</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEdit_ip">
     <property name="text">
      <string>192.168.10.207</string>
     </property>
     <property name="placeholderText">
      <string>IP地址</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_port">
     <property name="text">
      <string>8888</string>
     </property>
     <property name="placeholderText">
      <string>端口号</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_read"/>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_write"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_send">
       <property name="text">
        <string>send</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string>close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
